src/components/HeroHeader.jsx

```diff
@@ -7,7 +7,7 @@ import { withBreakpoints } from 'cozy-ui/transpiled/react'
 import { translate } from 'cozy-ui/react/I18n'
 import useInstanceSettings from 'hooks/useInstanceSettings'
 import useCustomWallpaper from 'hooks/useCustomWallpaper'
-import homeConfig from 'config/collect'
+import useAppDataset from 'hooks/useAppDataset'
```

Comment:
I discovered the useAppDataset hooks and I wonder if a hook was really necessary. It seems that for every component using this hook, we will have two renders where only one was necessary. Subsequent renders will not trigger a DOM access but still, it makes two render instead of one.

I wonder if the dataset access method should not be directly in CozyClient ? This way at app start, we init from the dataset div and then we can access values directly via CozyClient without a second render.

This should not be done in this PR since it is no the goal of this PR.

Reply:
I created an issue, better to comment there. https://github.com/cozy/cozy-home/issues/1395